server/ComputeNucleiFeatures/ComputeNucleiFeatures.xml

```diff
@@ -25,6 +25,46 @@
       <index>1</index>
       <description>Output HDF5 file</description>
     </file>
+    <boolean>
+      <name>size_shape_features</name>
+      <label>Size and Shape Features</label>
+      <channel>input</channel>
+      <longflag>size_shape</longflag>
+      <description>Compute Size and shape Features</description>
+      <default>true</default>
+    </boolean>
+    <boolean>
+      <name>fsd_features</name>
+      <label>Fourier Shape Descriptor Features</label>
+      <channel>input</channel>
+      <longflag>fsd</longflag>
+      <description>Compute Fourier Shape Descriptor Features</description>
+      <default>true</default>
+    </boolean>
+    <boolean>
+      <name>intensity_features</name>
+      <label>Intensity Features</label>
+      <channel>input</channel>
+      <longflag>intensity</longflag>
+      <description>Compute Intensity Features</description>
+      <default>true</default>
+    </boolean>
+    <boolean>
+      <name>gradient_features</name>
+      <label>Gradient/Edge Features</label>
+      <channel>input</channel>
+      <longflag>gradient</longflag>
+      <description>Compute Gradient/Edge Features</description>
+      <default>true</default>
+    </boolean>
+    <boolean>
+      <name>cytoplasm_features</name>
+      <label>Cytoplasm Features</label>
+      <channel>input</channel>
+      <longflag>cytoplasm</longflag>
+      <description>Compute Intensity and Gradient features from the cytoplasm channel</description>
+      <default>true</default>
+    </boolean>
```

Comment:
@cooperlab @slee172 Renamed FeatureExtraction CLI to ComputeNucleiFeatures and added boolean flags to toggle on/off the computation of each feature group